service/controller/resource/tccp/create.go

```diff
@@ -242,8 +242,14 @@ func (r *Resource) newParamsMain(ctx context.Context, cl apiv1beta1.Cluster, cr
 			return nil, microerror.Mask(err)
 		}
 
+		enableAWSCni := key.IsAWSCNINeeded(cl)
+		// If we have the aws-operator.giantswarm.io/legacy-aws-cni-pod-cidr, we still need to keep AWS cni subnets around.
+		if key.LegacyAWSCniCIDRBlock(cr) != "" {
+			enableAWSCni = true
```

Comment:
```suggestion
			keepAWSCni = true
```

Reply:
I'd rather change it to keep as mentioned in your comment

Reply:
this is not exactly true. The flag is actually used to enable aws CNI on v18, and incidentally reused to keep aws cni resources on v19. I think the name is correct (and it would require changing all cloudfront templates otherwise)

Reply:
Ok 👍🏻